lib/yeti_exception/version.rb

```diff
@@ -0,0 +1,4 @@
+module YetiException
+  # yeti_exception version
+  VERSION = "1.0.0"
```

Comment:
[nit] should this be lower or `pre` until you're ready for a real release version?

Reply:
Sure, I can do that until I'm ready to publish